leveldbjni/src/main/native-package/configure

```diff
@@ -662,6 +662,7 @@ JNI_EXTRA_CFLAGS
 JNI_JDK
 CXXCPP
 CPP
+LT_SYS_LIBRARY_PATH
```

Comment:
Do we need to include the change to this file? Is it automatically generated from autotool?

Reply:
https://github.com/utsaslab/leveldbjni/pull/1/files/4ece7fed84bed249e85ecc72d6f4e81d68a883b9#r219715836